code/obj/item/gun/energy.dm

```diff
@@ -1376,7 +1376,8 @@
 				are_you_the_law(M, msg[1])
 				return
 		else
-			are_you_the_law(M, msg[1])
+			if (!isflockmob(M)) //snowflake check because birds do not speak english
```

Comment:
why not check the language

Reply:
because the language wasn't actually being passed in before for some reason, have fixed it